.env.example

```diff
@@ -71,3 +71,6 @@ EMAIL_HOST_PASSWORD=
 # Use `sns` to have AWS SNS support. The AWS credentials must be present for this.
 # Use an empty string to log SMSs into a file instead. Recommended for development mode.
 SMS_DRIVER='sns'
+
+# Sentry Data Source Name
+SENTRY_DSN=
```

Comment:
need to update `docs/ENV.md`

Reply:
Man! Every time I think I didn't miss anything! Updating...